layers/+emacs/helpful/packages.el

```diff
@@ -0,0 +1,50 @@
+;;; packages.el --- helpful layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Johnson Denen <johnson@johnsons-macbook-pro.local>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; See the Spacemacs documentation and FAQs for instructions on how to implement
+;; a new layer:
+;;
+;;   SPC h SPC layers RET
+;;
+;;
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `helpful-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `helpful/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `helpful/pre-init-PACKAGE' and/or
+;;   `helpful/post-init-PACKAGE' to customize the package as it is loaded.
+
+;;; Code:
+
+(defconst helpful-packages
+  '(helpful))
+
+(defun helpful/init-helpful ()
+  (use-package helpful
+    :defer t
+    :init
+    (progn
+      (global-set-key (kbd "C-h f") #'helpful-callable)
+      (global-set-key (kbd "C-h k") #'helpful-key)
+      (global-set-key (kbd "C-h v") #'helpful-variable)
```

Comment:
Since this a layer, I don't think we should replace existing default help bindings because helpful bindings are provided as extra helpful functionality under  `SPC h h <key>`.  Just my opinion though.

Reply:
I'd like to hear more feedback regarding this. I debated it myself, but I ultimately concluded that if you use the `helpful` layer, you probably want it to replace the default help buffers.

If you would want the default help buffers, they'll still exist under `SPC h d`. But I'm open to hear more opinions.

Reply:
These keybidings should be under the Spacemacs leader key. That is how they are “supposed” to be accessed. Don’t mess with default bindings (especially since this is not how evil users would access it). I like @nixmaniack suggestion of using `SPC h h`, but if you feel like helpful should override the defaults they should be bound to `SPC h d`

Reply:
I have a custom function under `spc h h` that looks like it's very similar to what `spc h h .` does(without helpful though). I think this is a good thing to have, but the overrides should be a toggle option though.

Reply:
> but the overrides should be a toggle option though.

👍

Reply:
Thanks for the feedback, all.